src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java

```diff
@@ -27,53 +29,66 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MessagesConfigurator.class);
     private static GlobalProperties globalProperties = GlobalProperties.getInstance();
+    private static final String USER_ADMIN = "admin";
 
     private final LoginRestClient loginRestClient;
     private final MessagesPresentationRestClient messagesPresentationRestClient;
     private String rootEntitlementsAdmin = globalProperties.getString(PROPERTY_ROOT_ENTITLEMENTS_ADMIN);
 
     public void ingestConversations(String externalUserId) {
         int randomAmount = CommonHelpers
-            .generateRandomNumberInRange(globalProperties.getInt(PROPERTY_MESSAGES_MIN),
-                globalProperties.getInt(PROPERTY_MESSAGES_MAX));
-        IntStream.range(0, randomAmount).forEach(randomNumber -> {
+                .generateRandomNumberInRange(globalProperties.getInt(PROPERTY_MESSAGES_MIN),
```

Comment:
is it  possible to undo the additional indenting?

Reply:
yes